app/assets/stylesheets/views/_choose_sign_in.scss

```diff
@@ -0,0 +1,22 @@
+.verify-hint-box {
+  width: 100%;
+  border: 5px solid $govuk-border-colour;
+  padding: 15px;
+  box-sizing: border-box;
+  margin: -30px 0 30px;
+
+  .verify-hint-logos {
+    margin-bottom: 15px;
+
+    .verify-hint-logos-idp {
```

Comment:
We should use BEM for class names (e.g. `verify-hint-box`, `verify-hint-box__logos`, `verify-hint-box__logo-idp`, `.verify-hint-box__logo-verify`) which will make SCSS nesting unnecessary.